src/components/common/Footer/footer.scss

```
@@ -93,6 +93,11 @@
       }
     }
 
+    svg {
+      border: 1px solid #bbc1c6;
+      border-radius: 3px;
```

Some tokens I think are similar:

```suggestion
      border: var(--border-width-0) solid var(--color-neutral-3);
      border-radius: var(--border-radius-small);
```